src/components/ReactTags.tsx

```diff
@@ -133,11 +133,11 @@ const ReactTags = (props: ReactTagsProps) => {
     if (!enableFuzzySearch) {
       return;
     }
-    const suggestionList = suggestions.map((suggestion) => {
+    const suggestionList = props.suggestions.map((suggestion) => {
```

Comment:
Ahh this we should fix sometime, the state name and prop name being same can lead to lot of confusions